apps/user/src/user/dto/req.dto.ts

```diff
@@ -30,3 +30,23 @@ export class CreateUserDto {
   @IsNumber()
   public career_category_id: number;
 }
+
+export class UpdateUserDto {
+  @ApiProperty({ type: 'array', items: { type: 'number' } })
+  @IsArray()
+  @ArrayMaxSize(5)
+  @IsNumber({}, { each: true })
+  public stack_category_id?: number[];
+
+  @ApiProperty({ type: 'number' })
+  @IsNumber()
+  public position_category_id?: number;
+
+  @ApiProperty({ type: 'number' })
+  @IsNumber()
+  public project_category_id?: number;
```

Comment:
```suggestion
@ApiProperty({ type: 'array', items: { type: 'number' } })
  @IsArray()
  @ArrayMaxSize(3)
  @IsNumber({}, { each: true })
  public project_category_id?: number [];
```

micro Service 측의 req dto 도 앞서 말씀드린 리뷰처럼
project_cateogry_id가 number 배열을 받아오도록 변경 되어야 할 것 같습니다